Client/Client.Qt/Widgets/ChannelListWindow.cpp

```diff
@@ -28,7 +28,11 @@ ChannelListWindow::ChannelListWindow(std::shared_ptr<ListWidget>& anotherChannel
 
     _addChannelButton->setClickCallback([this]() { addChannelToMainChannelWidget(); });
     _updateChannelButton->setClickCallback([this]() { updateChannelListWindow(); });
-    ConnectionManager::getClient().askForChannelList();
+    if (ConnectionManager::isConnected())
+    {
+        ConnectionManager::getClient().askForChannelList();
+    
```

Comment:
Please, follow code style rules

Reply:
@Orange-hanter  You should point to the rule that you think was ignored.

Reply:
Chapter "Formatting" rule 22